src/jit/compiler.h

```diff
@@ -59,6 +59,7 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 
 #include "emit.h"
 
+#include "hwintrinsic.h"
```

Comment:
Since compiler.h includes gentree.h, which includes hwintrinsic.h, this doesn't seem necessary.

Reply:
👍 I'll try removing and it see if it still compiles

Reply:
It's `gentree.cpp` which includes `hwintrinsic.h`, not `gentree.h`, so we still need this for now.